src/main/webapp/app/routes.tsx

```diff
@@ -29,6 +31,8 @@ const Routes = () => (
       <ErrorBoundaryRoute exact path="/auth/email/login" component={AuthEmailLogin} />
       <ErrorBoundaryRoute exact path="/auth/email/register" component={AuthEmailRegister} />
       <ErrorBoundaryRoute exact path="/auth/email/reset" component={AuthEmailReset} />
+      <ErrorBoundaryRoute exact path="/events/upcoming" component={Events} />
+      <ErrorBoundaryRoute exact path="/events/:id/details" component={EventDetails} />
```

Comment:
There is a path defined in index.tsx of entity/event directory i think

Reply:
Resolve in 19e9cf4e40e2cb39d14dcee1fa6d29764e323827